chef/cookbooks/nova/recipes/compute.rb

```diff
@@ -154,7 +154,8 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
 
       if node[:nova][:libvirt_type] == "kvm"
         if node.platform_version.to_f >= 12.0
-          package "qemu-kvm"
+          package "qemu-kvm" if node[:architecture] == "x86_64"
+          package "qemu-ppc" if node[:architecture] =~ "ppc64le"
```

Comment:
I think we should say just package "qemu" and let zypper decide what subpackages to install. Since qemu should suggest qemu-ppc on power and qemu-x86 on x86.

IIRC qemu-kvm is just a compatibility package which provides qemu-kvm wrapper.